fulltext/__init__.py

```diff
@@ -226,12 +224,25 @@ def _get_file(backend, f, **kwargs):
             return backend._get_path(t.name, **kwargs)
 
 
-def get_backend_mod(ext):
-    if not ext.startswith('.'):
+def backend_from_mime(mime):
+    try:
+        mod_name = MIMETYPE_TO_BACKENDS[mime]
+    except KeyError:
+        mimebin = 'application/octet-stream'
```

Comment:
Maybe define this as a "const" `DEFAULT_MIME = 'application/octet-stream'`

Reply:
sounds good